xml/kvm_virtualization_basics.xml

```diff
@@ -18,12 +18,12 @@
   <title>Basic Components</title>
 
   <para os="sles">
-   &kvm; is a full virtualization solution for the &x86-64; and the
+   &kvm; is a full virtualization solution for the &x86-64;, &aarch64; and the
    &zseries; architectures supporting hardware virtualization.
   </para>
 
   <para os="osuse">
-   &kvm; is a full virtualization solution for the &x86-64; architecture
+   &kvm; is a full virtualization solution for the &x86-64;, &aarch64 and the &zseries; architectures
```

Comment:
The semicolon after &aarch64 is missing. Correct is:
```suggestion
   &kvm; is a full virtualization solution for the &x86-64;, &aarch64; and the &zseries; architectures
```
I would not list z Systems as long as we don't release openSUSE for s390x.